experimental/uefi/proto/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "uefi-proto"
```

Comment:
nit: Is there not a better potential name? Calling the directory `proto` leads to a directory tree that contains `proto/proto` to store the proto files.

Reply:
Turns out the crate was unnecessary.